packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -92,24 +91,26 @@ class AndroidView extends StatefulWidget {
 
   /// Which gestures should be forwarded to the Android view.
   ///
-  /// The gesture recognizers on this list participate in the gesture arena for each pointer
-  /// that was put down on the widget. If any of the recognizers on this list wins the
+  /// The gesture recognizers built by factories in this set participate in the gesture arena for
+  /// each pointer that was put down on the widget. If any of these recognizers win the
   /// gesture arena, the entire pointer event sequence starting from the pointer down event
   /// will be dispatched to the Android view.
   ///
+  /// When null an empty set of gesture recognizer factories is used, and a pointer event sequence
+  /// wil only be dispatched to the Android view if no other member of the arena claimed it.
```

Comment:
wil -> will

also, this whole sentence makes no sense. maybe the "and" is extraneous?

Reply:
rephrased a bit, is it more clear now?